demo-site/scripts/create_static_files.js

```diff
@@ -0,0 +1,105 @@
+//region imports
+const fs = require('fs');
```

Comment:
This may not be necessary but it can stay because we still have to generate the static files based on the app_id and first_party configs so it's okay for now.